localstack-core/localstack/services/lambda_/event_source_mapping/esm_event_processor.py

```diff
@@ -44,11 +46,13 @@ def process_events_batch(self, input_events: list[dict]) -> None:
                 logLevel=LogLevel.TRACE,
             )
             # Target Stage
-            self.process_target_stage(events)
+            payload = self.process_target_stage(events)
```

Comment:
This is for future reference if we need to do anything with the returned payload.